server/chat-commands/info.js

```diff
@@ -2339,6 +2339,19 @@ const commands = {
 	},
 	pickrandomhelp: [`/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements.`],
 
+	'!shuffle': true,
+	shuffle(target, room, user) {
+		if (!target) return false;
```

Comment:
This check is counterproductive with the line after.

Reply:
mb, fixed